accepted/future-releases/enhanced-enums/feature-specification.md

```diff
@@ -197,7 +197,7 @@ The `enum` declaration:
 enum LogPriority with LogPriorityMixin implements Comparable<LogPriority> {
   warning(2, "Warning"),
   error(1, "Error"),
-  log.unknown("Log"),
+  LogPriority.unknown("Log"),
```

Comment:
Isn't this an enum value declaration? In that case it looks correct to have a name like `log`, but a value named `LogPriority` would conflict with the name of the enclosing class. So it seems likely that this change should not be performed.

Reply:
Thank you. This change was wrong. Reverted